modules/distr-tracing-rn-new-features.adoc

```diff
@@ -17,25 +17,125 @@ This release adds improvements related to the following components and concepts.
 
 This release of {DTProductName} addresses Common Vulnerabilities and Exposures (CVEs) and bug fixes.
 
+=== {JaegerName}
 
+==== Bug fixes
 
+* Expose Jaeger Query gRPC port (16685) on the Jaeger Query service.
 
+// link:https://issues.redhat.com/browse/TRACING-3322[TRACING-3322]
 
+* When deploying Service Mesh on SNO in a disconnected environment , the Jaeger Pod frequently goes into Pending state.
 
+// link:https://issues.redhat.com/browse/TRACING-3312[TRACING-3312]
 
+* Wrong port is exposed for jaeger-production-query resulting in connection refused.
+Fixed by exposing Jaeger Query gRPC port (16685) on the Jaeger Query deployment.
 
+// link:https://issues.redhat.com/browse/TRACING-2968[TRACING-29638]
 
+* Jaeger operator pod restarting with OOMKilled with the default memory value.
+Fixed by removing resource limits.
 
+// link:https://issues.redhat.com/browse/TRACING-3173[TRACING-3173]
 
+=== {TempoName}
 
+The {TempoName} is Technology Preview feature for {DTProductName}.
 
+==== Enhancements
 
+* Support the operator level 4 (Deep Insights).
+This feature enables upgrade, monitoring and alerting of `+TempoStack+` instances and operator.
+* Add Ingress and OpenShift Route configuration for the Gateway.
+* Support managed and unmanaged state in the `+TempoStack+` custom resource.
+* Expose additional ingestion protocols (Jaeger Thrift binary, Jaeger Thrift compact, Jaeger gRPC and Zipkin) in the Distributor service.
+When the Gateway is enabled only OTLP gRPC is enabled.
+* Expose Jaeger Query gRPC endpoint on the Query Frontend service.
+* Support multitenancy without the Gateway (without authentication and authorization).
 
+==== Bug fixes
 
+* Support disconnected environments.
 
+// link:https://issues.redhat.com/browse/TRACING-3145[TRACING-3145]
 
+* Enable mTLS communication between Tempo components.
 
+// link:https://issues.redhat.com/browse/TRACING-3091[TRACING-3091]
 
+* Remove resource limits for Tempo operator.
+
+// link:https://issues.redhat.com/browse/TRACING-3204[TRACING-3204]
+
+==== Known issues
+
+* Custom TLS CA option is not implemented for connecting to an object storage.
+
+// link:https://issues.redhat.com/browse/TRACING-3462[TRACING-3462]
+
+* When you use the Jaeger user interface (UI) with Tempo Operator, the Jaeger UI lists only services that have sent traces within the last 15 minutes.
+For services that have not sent traces within the last 15 minutes, those traces are still stored even though they are not visible in the Jaeger UI.
+
+// link:https://issues.redhat.com/browse/TRACING-3139[TRACING-3139]
+
+* Tempo query frontend service should not use internal mTLS when gateway is not deployed. This issue does not affect Jaeger Query API. The workaround is to disable mTLS.
+
+.Disable mTLS
```

Comment:
@rubenvp8510 could you please review the procedure to disable the mTLS? It ends on line 110.

Reply:
Responded by Slack.

Reply:
This procedure looks good!

Reply:
Verified the steps to disable the mTLS.